File: ICSharpCode.Decompiler.Tests/TestCases/VBPretty/Select.cs
using Microsoft.VisualBasic.CompilerServices;
using System;

[StandardModule]
internal sealed class Program
{
	public static void SelectOnString()
	{
		switch (Environment.CommandLine) {
			case "123":
				Console.WriteLine("a");
				break;
			case "444":
				Console.WriteLine("b");
				break;
			case "222":
				Console.WriteLine("c");
				break;
			case "11":
				Console.WriteLine("d");
				break;
			case "dd":
				Console.WriteLine("e");
				break;
			case "sss":
				Console.WriteLine("f");
				break;
			case "aa":
				Console.WriteLine("g");
				break;
			case null:
			case "":
				Console.WriteLine("empty");
				break;
		}
	}
}

File: ICSharpCode.Decompiler.Tests/TestCases/VBPretty/Select.vb
Imports System

Module Program
	Sub SelectOnString()
		Select Case Environment.CommandLine
			Case "123"
				Console.WriteLine("a")
			Case "444"
				Console.WriteLine("b")
			Case "222"
				Console.WriteLine("c")
			Case "11"
				Console.WriteLine("d")
			Case "dd"
				Console.WriteLine("e")
			Case "sss"
				Console.WriteLine("f")
			Case "aa"
				Console.WriteLine("g")
			Case ""
				Console.WriteLine("empty")
		End Select
	End Sub
End Module
